src/background.ts

```diff
@@ -560,6 +562,59 @@ chrome.commands.onCommand.addListener(async (command: string) => {
       });
       break;
 
+    case "autofill":
+      await new Promise(
+        (resolve: () => void, reject: (reason: Error) => void) => {
+          try {
+            return chrome.tabs.executeScript(
+              { file: "/dist/content.js" },
+              () => {
+                chrome.tabs.insertCSS({ file: "/css/content.css" }, resolve);
+              }
+            );
+          } catch (error) {
+            console.error(error);
+            return reject(error);
+          }
+        }
+      );
+
+      if (cachedPassphrase === "") {
+        return;
+      }
+
+      chrome.tabs.query(
+        { active: true, lastFocusedWindow: true },
+        async (tabs) => {
+          const tab = tabs[0];
+          if (!tab || !tab.id) {
+            return;
+          }
+          contentTab = tab;
+
+          const siteName = await getSiteName();
+          const entries = await EntryStorage.get();
+          const matchedEntries = getMatchedEntries(siteName, entries);
+
+          if (matchedEntries && matchedEntries.length === 1) {
```

Comment:
matchedEntries.length <= 1

Reply:
if `matchedEntries.length == 0`, `const entry = matchedEntries[0]` will be `undefined`. We should auto fill code when there is only one matched entry

Reply:
Sorry, incorrect logic on my end. I meant `matchedEntries.length >= 1`. It won't past a code if there is more than one match.

Reply:
I agree, that would be the best solution.

Reply:
Birden fazla eşleşen giriş varsa, kullanıcıdan her zaman doğru olanı seçmesini istememiz gerektiğini düşünüyorum, ancak ilkini değil. Ne düşünüyorsun?